### ISSUE TITLE
Cannot install on Plesk.

### ISSUE THREAD

**Issue by Ratingthomas** [mosparo/mosparo#57]
When I try to is install mosparo on Plesk. I get a internal server error.

In the logs I get the following error
```
[2023-05-09T06:48:52.533099+00:00] request.CRITICAL: Uncaught PHP Exception RuntimeException: "Session Storage was not able to create directory "/var/lib/php/sessions"." at /var/www/vhosts/DOMAIN/example.DOMAIN/vendor/symfony/http-foundation/Session/Storage/Handler/NativeFileSessionHandler.php line 49 {"exception":"[object] (RuntimeException(code: 0): Session Storage was not able to create directory \"/var/lib/php/sessions\". at /var/www/vhosts/DOMAIN/example.DOMAIN/vendor/symfony/http-foundation/Session/Storage/Handler/NativeFileSessionHandler.php:49)"} []
```

**Comment by zepich** [comment]
Hi @Ratingthomas 

Thank you very much for your issue.

I can reproduce it. It is because of the session table in v0.3.9.

I will check how we can solve this issue.

Kind regards,

zepich

**Comment by zepich** [comment]
### A workaround for the moment:

1. Edit the file `config/packages/framework.yaml`
2. Modify the line `handler_id`. Set the value to `null`
```
...
    session:
        handler_id: null #Mosparo\Session\DatabaseSessionHandler
...
```
3. Save the file
4. Delete the directory `prod` in the path `var/cache`
5. Use the installer in the web browser
6. After mosparo is set up, go back end edit the file `config/packages/framework.yaml` again
7. Change back to the original value
```
...
    session:
        handler_id: Mosparo\Session\DatabaseSessionHandler
...
```
8. Delete the directory `prod` in the path `var/cache` (again)
9. Use mosparo as usually

**Comment by Ratingthomas** [comment]
Hi @zepich ,

I tried your workaround and it installs and works correctly now. Thank you!

**Comment by zepich** [comment]
Hi @Ratingthomas 

Thank you very much for your feedback.

We've just released v0.3.17, which contains a fix for this issue, so the workaround is no longer needed on Plesk hosting (or other similar hostings).

Kind regards,

zepich
